core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/model/AbstractCamelModelElement.java

```diff
@@ -971,13 +971,18 @@ private void updateElementDescription(String name, Object value, Element e) {
 	 * @param e
 	 */
 	private void updateAttribute(String name, Object newValue, Object oldValue, Element e) {
-		String defaultValue = this.underlyingMetaModelObject != null
-				? this.underlyingMetaModelObject.getParameter(name).getDefaultValue() : null;
-		if (defaultValue != null && defaultValue.equals(getMappedValue(newValue))) {
-			// default value -> no need to explicitly set it -> delete
-			// existing
-			e.removeAttribute(name);
-		} else {
+// 
```

Comment:
Yeah, I asked several people and I got different opinions on that. We can discuss that later.